src/main/java/com/uber/cadence/internal/sync/WorkflowThreadImpl.java

```diff
@@ -220,18 +224,31 @@ public boolean isCancelRequested() {
 
   @Override
   public void start() {
+    int attempts = 2;
+    for (int i = 0; i < attempts; i++) {
```

Comment:
Refactor to eliminate the loop to simplify the code

Reply:
replaced with two try statements